enhancements/machine-config/mco-cgroupsv2-support.md

```diff
@@ -40,12 +43,10 @@ status: implementable
 
 ## Summary
 
-Control Group v2 (cgroup v2) enablement in Kubernetes has progressed to beta
+Control Group v2 (cgroup v2) enablement in Kubernetes has progressed to stable
 [upstream](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2254-cgroup-v2).
-The underlying runtime (cri-o) and supporting subsystems are now ready for
-customers to begin their own testing with it. Not all workloads will be
-compatible with cgroup v2, so it will *not* be enabled by default within
-OpenShift at this time.
+cgroup v2 is enabled by default in the freshly installed Openshift clusters >= 4.14.
+Control Group v1 is a deprecated feature from OCP 4.16 and the support is intended to be removed from OCP >= 4.19
```

Comment:
so to clarify, we'd backport the upgradeability check to 4.18? Assuming we don't make GA we'd be requiring all clusters to upgrade to some 4.18.z with this check before proceeding to 4.19?

Reply:
For all the clusters that are on `CgroupMode_V1`, We would set the `Upgradeblity` of the cluster operator to `False`.
Thus, preventing an upgrade requires the customer to manually set the `CgroupMode` to `CgroupMode_V2`